extensions/amp-script/0.1/amp-script.js

```diff
@@ -71,13 +74,29 @@ export class AmpScript extends AMP.BaseElement {
         isLayoutSizeDefined(layout);
   }
 
-  /** @override */
-  layoutCallback() {
+  /** @return {!Promise<boolean>} */
+  isExperimentOn_() {
     if (!isExperimentOn(this.win, 'amp-script')) {
-      user().error(TAG, 'Experiment "amp-script" is not enabled.');
-      return Promise.reject('Experiment "amp-script" is not enabled.');
+      return Promise.resolve(false);
```

Comment:
Is this right? Shouldn't it be `if (isExperimentOn(...)) {return Promise.resolve(true)}`? I.e. either the experiment is enabled manually for local dev testing or via origin trial?

Reply:
Oops, good catch. Writing code in conference is not best.